server/src/main/kotlin/com/cuhacking/watershed/resources/AuthResource.kt

```diff
@@ -0,0 +1,48 @@
+package com.cuhacking.watershed.resources
+
+import com.cuhacking.watershed.db.Database
+import com.cuhacking.watershed.util.JwtManager
+import io.ktor.application.*
+import io.ktor.auth.*
+import io.ktor.http.*
+import io.ktor.request.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.Serializable
+import org.mindrot.jbcrypt.BCrypt
+import javax.inject.Inject
+
+@Serializable
+data class InputCredentials(val email: String, val password: String)
+
+class AuthResource @Inject constructor(private val database: Database,
+                                       private val dispatcher: CoroutineDispatcher,
+                                       private val jwtManager: JwtManager) {
+    fun Routing.routing() {
+        route("/login") {
```

Comment:
Aren't these `route` calls redundant? Could we just call `post("/login")` instead? We could reserve `route` calls for instances where an endpoint has multiple methods associated with it.